jobs/service-fabrik-broker/templates/config/settings.yml.erb

```diff
@@ -29,7 +29,7 @@ production:
   enable_circuit_breaker: <%= p('broker.enable_circuit_breaker') %>
   feature:
     ServiceInstanceAutoUpdate: <%= p('broker.feature.ServiceInstanceAutoUpdate') %>
-
+  http_timeout: 120000
```

Comment:
value cannot be hard code. Please define this in spec and define a default value from spec.